docs/csharp/how-to/search-strings.md

```diff
@@ -0,0 +1,67 @@
+---
+title: "How to: search strings (C# Guide)"
+ms.date: 02/21/2018
+ms.prod: .net
+ms.technology: 
+  - "devlang-csharp"
+ms.topic: "article"
+helpviewer_keywords: 
+  - "searching strings [C#]"
+  - "strings [C#], searching with String methods"
+  - "strings [C#], searching with regular expressions"
+ms.assetid: fb1d9a6d-598d-4a35-bd5f-b86012edcb2b
+ms.author: "wiwagn"
+---
+
+# How to: search strings
+
+You can use two main strategies to search for text in strings. Methods of the <xref:System.String> class search for specific text. Regular expressions search for patterns in text.
+
+[!INCLUDE[interactive-note](~/includes/csharp-interactive-note.md)]
+
+The [string](../language-reference/keywords/string.md) type, which is an alias for the <xref:System.String?displayProperty=nameWithType> class, provides a number of useful methods for searching the contents of a string. Among them are <xref:System.String.Contains%2A>, <xref:System.String.StartsWith%2A>, <xref:System.String.EndsWith%2A>, <xref:System.String.IndexOf%2A>, <xref:System.String.LastIndexOf%2A>. The <xref:System.Text.RegularExpressions.Regex?displayProperty=nameWithType> class provides a rich vocabulary to search for patterns in text. In this article, you learn these techniques and how to choose the best method for your needs.
+
+## Does a string contain text?
+
+The <xref:System.String.Contains%2A?displayProperty=nameWithType>, <xref:System.String.StartsWith%2A?displayProperty=nameWithType> and <xref:System.String.EndsWith%2A?displayProperty=nameWithType> methods search a string for specific text. The following example shows each of these methods, and a variation that uses a case insensitive search:
```

Comment:
nit: either no comma before "and", or change the last phrase to read "and a variation uses..."